common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAccount.java

```diff
@@ -367,6 +368,11 @@ public String getAvatarUrl() {
         return SchemaUtil.getAvatarUrl(mIDToken);
     }
 
+    @Override
+    public String getClientInfo() {
```

Comment:
Should this be renamed to `getRawClientInfo()` in the base class?  As by naming convention in this class `getClientInfo ` should ideally return `ClientInfo`

Reply:
Sounds good. Thanks Brian 👍